content/index.md

```diff
@@ -61,9 +61,13 @@ The Multiformats project began through [the IPFS Project](https://ipfs.io). It i
 
 <ul class="project-list">
 	<li>
-		<a href="https://ipfs.io">
+		<a href="https://ipfs.tech">
 		<img src="projects/ipfs.png" />IPFS</a>
-		- a peer-to-peer hypermedia protocol and distributed file system.
+		- an open system to manage data without a central server.
+	</li>
+	<li>
+		<a href="https://github.com/multiformats/cid"></a>
+		- self-describing content-addressed identifiers for distributed systems.
```

Comment:
```suggestion
		- a self-describing content-addressed identifier scheme for distributed systems, used to address files in IPFS and systems interoperable with it, as well as in other systems like IPLD, where it is the primary way of expressing links between documents.
```

Reply:
(probably more verbose than it makes sense to be here, but just proposing the maximally-detailed version of the summary of what a CID is :D )

Reply:
Great idea!